ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityEmailQuestionBlock.ts

```diff
@@ -0,0 +1,23 @@
+import { ActivityTextQuestionBlock } from './activityTextQuestionBlock';
+
+export class ActivityEmailQuestionBlock extends ActivityTextQuestionBlock {
+  public email: string | null = null;
+  public emailConfirmation: string | null = null;
+  constructor() {
+    super();
+  }
+
+  get answer(): string | null {
+    return this.email;
+  }
+
+  set answer(value: string | null) {
+    this.email = this.emailConfirmation = value;
+    this.validate();
+  }
```

Comment:
If we do set a value for the block, we are going to start off email and confirm value being the same. Only the component that works with emails will care.